trlx/trainer/accelerate_base_trainer.py

```diff
@@ -474,12 +480,23 @@ def learn(self):  # noqa: C901
                     # gradient update per batch, PPO for example commonly performs
                     # multiple gradient updates on the same batch of data.
                     # https://arxiv.org/pdf/1707.06347.pdf
-                    forward_time = time()
-                    loss, stats = self.loss(batch)
-                    forward_time = time() - forward_time
-                    backward_time = time()
-                    self.accelerator.backward(loss)
-                    backward_time = time() - backward_time
+                    forward_time = 0
+                    backward_time = 0
+                    stats_accum = []
+                    for mbi in range(self.num_mb):
+                        forward_time -= time()
+                        loss, stats = self.loss(batch)
```

Comment:
Shouldn't this get loss using a minibatch sliced from `batch`?

Reply:
Ah yeah good catch (I copied this over from a different branch and forgot to change this)